.puppet-lint.rc

```diff
@@ -0,0 +1,4 @@
+--no-arrow_alignment-check
+--no-double_quoted_strings-check
+--no-2sp_soft_tabs-check
+--no-documentation-check
```

Comment:
I'd rather put the arguments inside `tools/lint`; it's too easy to not know these exist here.